scss/layout/_panel.scss

```diff
@@ -75,28 +80,33 @@ $block: #{$fd-namespace}-panel;
         }
     }
     &__filters {
+        @include fd-reset();
         padding: $fd-panel-filters-padding;
         //border-bottom: solid 1px $fd-panel-filters-border-color;
         border-bottom-style: solid;
         border-bottom-width: 1px;
         @include fd-var-color("border-bottom-color", $fd-panel-filters-border-color, --fd-panel-divider-color);
         transition: all $fd-panel-fiters-transition-params;
-        &.is-hidden,
-        &[aria-hidden="true"] {
+        &.is-hidden {
           padding-top: 0;
           padding-bottom: 0;
           max-height: 0;
           overflow: hidden;
         }
     }
     &__body {
+        @include fd-reset();
         padding: $fd-panel-padding;
         &--full-bleed,
         &--bleed {
           padding: 0;
         }
+        p {
+            @include fd-reset-block();
+        }
```

Comment:
The more I look at this one, the more I don't like it.  A panel, like other container components, should not really have any rules about _what_ content can appear in its body.  Because of that, we shouldn't be handling any child element styling here.  This example if for a `<p>`, but what I the consumer put a `<ul>` (or any other element) in the body?

Reply:
Actually, how about we remove this `p` block (and the `fd-reset-block` mixin) and replace it with this (like it's being done for modals):
```scss
    > :first-child {
      margin-top: 0;
    }

    > :last-child {
      margin-bottom: 0;
    }
```
This should accomplish the same thing, but not lock us down to individual elements.